Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.10.8-slim
+LABEL maintainer="yaroslavbordovoy@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR app/
```

Comment:
The `WORKDIR` path should be `WORKDIR app` instead of `WORKDIR app/` to ensure proper directory setup.